packages/next/src/client/index.tsx

```diff
@@ -509,6 +509,7 @@ function renderReactElement(
   if (!reactRoot) {
     // Unlike with createRoot, you don't need a separate root.render() call here
     reactRoot = ReactDOM.hydrateRoot(domEl, reactEl, {
+      // @ts-expect-error Missing errorInfo in @types/react
```

Comment:
Need to double check whether this is part of stable release channels or next or experimental.